packages/notifi-react-card/lib/components/subscription/EventTypeHealthCheckRow.tsx

```diff
@@ -53,74 +66,120 @@ export const EventTypeHealthCheckRow: React.FC<
     ratios = config.checkRatios.value;
   }
 
-  const enabled = !loading && !!alerts[alertName];
-
   useEffect(() => {
-    if (isValueType) {
-      setSelectedRatio(ratios[-1]?.ratio);
+    if (loading) {
+      return;
+    }
+    const alert = alerts[alertName];
+    const checkRatios = ratios.map((ratio) => ratio.ratio);
+    if (alert) {
+      const alertRatioValue = JSON.parse(alert.filterOptions ?? '').threshold;
```

Comment:
JSON.parse will throw an error if you use `''` I think.

Let's add some safety around parsing the filterOptions